docs/en/latest/health-check.md

```diff
@@ -26,8 +26,13 @@ title: Health Check
 Health Check of APISIX is based on [lua-resty-healthcheck](https://github.com/Kong/lua-resty-healthcheck),
 you can use it for upstream.
 
-Note that we only start the health check when the upstream is hit by a request.
+Note:
+
+* We only start the health check when the upstream is hit by a request.
 There won't be any health check if an upstream is configured but isn't in used.
+* If there is not healthy node can be chosen, we will continue to access the upstream.
+* We won't start the health check when the upstream only has one node, as we will access
+it whether this unique node is healthy or not.
```

Comment:
```suggestion
it no matter this unique node is healthy or not.
```

Reply:
`whether ... or not` is equal to `no matter`.